_summaries/zhao2020point_2.md

```diff
@@ -1,19 +1,68 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
----
+title: Point Transformer
+paper: Zhao2020point
+author: # sritank
+score: # 9/10
+
+* Core idea?
+
+They design a Point Transformer layer for point cloud processing suitable for point cloud labeling and segmentation. They introduce vector self-attention and trainable position encoding among others to make their residual transformer network invariant to permutation and cardinality. 
+
+* Technical Implementation?
+
+Given a set of feature vectors {x_i}, the vector self-attention function can be written as below:
+
+![Vector attention](zhao2020point_2a.png)
+
+where alpah(x) is the transformed features and gamma is the attention vector. They apply self attention locally with a neighbourhood of each datapoint in the cloud. The point transformer layer is created as follows:
+
+![Point transformer layer](zhao2020point_2b.png)
+
+They define their position encoding based on the point coordinates multiplied with trainable parameters as below:
+
+![Position encoding](zhao2020point_2c.png)
```

Comment:
It might be a good idea to mention that the trainable position encoding is added to both the attention and feature transformation branches of the vector self-attention formula.